src/map/npc.c

```diff
@@ -4391,7 +4398,7 @@ int npc_parsesrcfile(const char* filepath, bool runOnInit)
 			}
 		}
 
-		if( strcasecmp(w2,"warp") == 0 && count > 3 )
+		if( strncasecmp("warp", w2, 4) == 0 && count > 3 )
```

Comment:
this would also allow "warp5000" as definition, because it will only validate the first 4 characters.
`( strcasecmp(w2,"warp") == 0 || strcasecmp(w2,"warp2") == 0 ) && count > 3`